docs/HISTORY.txt

```diff
@@ -4,7 +4,10 @@ Changelog
 2.4.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Redefine used ftw.mobile scss variables.
```

Comment:
@raphael-s could you also rephrase the changelog entry? We are no longer redefining any variables.
Actually we just prevent loading the ftw.mobile-customizations when ftw.mobile is not installed, awoiding compilation errors.